build.mjs

```diff
@@ -20,7 +20,7 @@ async function esbuildBrowserExtension() {
         minify: true,
         legalComments: 'none',
         sourcemap: true,
-        plugins: [inlineImage()],
+        // plugins: [inlineImage()],
```

Comment:
Why remove this plugin? This will cause packaging failure due to the inability to package gif images.

Reply:
chrome extension，logo image load failure.

Reply:
@100apps Thank you very much for your help in finding this problem and alerting me to it, it was a problem caused by me, I have fixed the problem and pushed to the main branch, I may need to ask you to change this line back and sync the main branch, sorry

Reply:
@

> @100apps Thank you very much for your help in finding this problem and alerting me to it, it was a problem caused by me, I have fixed the problem and pushed to the main branch, I may need to ask you to change this line back and sync the main branch, sorry

done. thanks